src/Menu/Menu.tsx

```diff
@@ -61,6 +61,16 @@ export interface MenuProps {
    */
   maxHeight: string;
 
+  /**
+   * Popper.js Position modifiers.
+   */
+  modifiers?: any;
+
+  /**
+   * Whether the menu should be the same width as the reference element
+   */
+  sameReferenceWidth?: boolean;
```

Comment:
This is too wordy - i think `autoWidth` would be better. Also we need to add `minWidth` and `maxWidth` if we are going to do this.

```suggestion
  autoWidth?: boolean;
```